server/build.gradle

```diff
@@ -190,6 +190,7 @@ dependencies {
   compileOnly project(':app-server')
   compileOnly group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.47'
   compileOnly group: 'org.eclipse.jetty.websocket', name: 'websocket-server', version: versions.jetty
+  compile group: 'org.eclipse.jetty.websocket', name: 'javax-websocket-server-impl', version: versions.jetty
```

Comment:
@ketan just to confirm, the websocket jars (jetty's impl + jsr-356 api) only need to be in the `jetty9` and `development-server` modules? These do not need to be in `server`?